src/internal/observable/combineLatest.ts

```diff
@@ -44,19 +44,19 @@ export function combineLatest<O1 extends ObservableInput<any>, O2 extends Observ
 export function combineLatest<O1 extends ObservableInput<any>, O2 extends ObservableInput<any>, O3 extends ObservableInput<any>, O4 extends ObservableInput<any>, O5 extends ObservableInput<any>, O6 extends ObservableInput<any>, R>(v1: O1, v2: O2, v3: O3, v4: O4, v5: O5, v6: O6, resultSelector: (v1: ObservedValueOf<O1>, v2: ObservedValueOf<O2>, v3: ObservedValueOf<O3>, v4: ObservedValueOf<O4>, v5: ObservedValueOf<O5>, v6: ObservedValueOf<O6>) => R, scheduler?: SchedulerLike): Observable<R>;
 
 // With a scheduler (deprecated)
-/** @deprecated Passing a scheduler here is deprecated, use {@link subscribeOn} and/or {@link observeOn} instead */
+/** @deprecated The scheduler argument is deprecated, use {@link subscribeOn} and/or {@link observeOn}. Details: {@link https://rxjs.dev/deprecations/scheduler-argument} */
```

Comment:
Is the content of the `deprecated` tag shown anywhere in the docs? AFAICT, it isn't. If that is the case, can we please remove this `{@link ... }` business, as it just makes the messages harder to read in linter reports and in the VS Code hints.

Reply:
I changed it for now, but in the long run I would like to have this in the docs as well. 
In the long run I would like to have this in the docs and IntelliJ is supporting @link and @see, so it's mainly just vscode that has several open issues for that:
https://github.com/microsoft/TypeScript/issues/35524
https://github.com/microsoft/TypeScript/issues/16498

As soon as this is supported by VsCode I'd like to change it to @link etc again